Product/Production/Common/CONNECTCoreLib/src/main/java/gov/hhs/fha/nhinc/util/HomeCommunityMap.java

```diff
@@ -57,6 +57,7 @@
     private static final Logger LOG = LoggerFactory.getLogger(HomeCommunityMap.class);
     private static ConnectionManagerCache connection = ConnectionManagerCache.getInstance();
     private static PropertyAccessor propertyAccessor = PropertyAccessor.getInstance();
+    private static final String URN_OID_PREFIX = "urn:oid:";
```

Comment:
Instead of introducing a new constant we can use existing constant from NhincConstants.java called - HCID_PREFIX